packages/aml-checklist/src/core/index.tsx

```diff
@@ -36,27 +35,34 @@ export const renderApp = (Component: React.ComponentType) => {
 }
 
 const run = async () => {
-  await fetch('config.json')
-    .then(response => response.json())
-    .then((config: Config) => {
-      window.reapit.config = config
-      const isLocal = config.appEnv === 'local'
-      if (!isLocal && config.sentryDns) {
-        Sentry.init({
-          release: process.env.APP_VERSION,
-          dsn: config.sentryDns,
-          environment: config.appEnv,
-        })
-      }
-      if (!isLocal && config.googleAnalyticsKey) {
-        ReactGA.initialize(config.googleAnalyticsKey)
-        ReactGA.pageview(window.location.pathname + window.location.search)
-      }
-      renderApp(App)
-    })
-    .catch(error => {
-      console.error('Cannot fetch config', error)
-    })
+  try {
+    const configRes = await fetch('config.json')
+    const config = (await configRes.json()) as Config
+    const isLocal = config.appEnv === 'local'
+
+    if (!isLocal && config.sentryDns) {
+      Sentry.init({
+        release: process.env.APP_VERSION,
+        dsn: config.sentryDns,
+        environment: config.appEnv,
+      })
+    }
+
+    if (!isLocal && config.googleAnalyticsKey) {
+      ReactGA.initialize(config.googleAnalyticsKey)
+      ReactGA.pageview(window.location.pathname + window.location.search)
+    }
+
+    window.reapit.config = config
+
+    // I import the app dynamically so that the config is set on window and I avoid any
+    // runtime issues where config is undefined
+    const { default: App } = await import('./app')
+
+    renderApp(App)
+  } catch (error) {
+    console.error('Cannot fetch config', error)
```

Comment:
LGTM. This error seems geneirc. At this error in this big try catch could belong to other reasons that not involve fetching `config.json`.  E.g.: ReactGA is failed to initialize and throw

Reply:
TBH, it's just the error that was there before, I'll have a quick look